democracy_club/apps/projects/templates/projects/candidates.html

```diff
@@ -1,17 +1,18 @@
 {% extends "projects/projects_base.html" %}
 {% load markdown_deux_tags %}
 {% load static %}
+{# fmt:off #}
+
 {% block title %}
 Candidates Wiki
 {% endblock %}
 
 {% block content %}
 <div class="ds-stack-smaller">
     <h1>Candidates Wiki</h1>
-    <img src="{% static "ynr-on-a-phone.png" %}">
+    <img src="{% static 'ynr-on-a-phone.png' %}" alt="">
```

Comment:
Would be better for accessibility to add some content to the `alt` tags?

Reply:
Absolutely. I plan to do an accessibility audit in another PR, but I'll fix this one here.